omnigibson/object_states/overlaid.py

```diff
@@ -0,0 +1,66 @@
+from omnigibson.object_states.kinematics import KinematicsMixin
+from omnigibson.object_states.object_state_base import BooleanState, RelativeObjectState
+from omnigibson.object_states.touching import Touching
+from omnigibson.utils.constants import PrimType
+import omnigibson.utils.transform_utils as T
+from omnigibson.macros import create_module_macros
+
+from scipy.spatial import ConvexHull, HalfspaceIntersection
+from scipy.spatial.qhull import QhullError
+import numpy as np
+import trimesh
+import itertools
+
+# Create settings for this module
+m = create_module_macros(module_path=__file__)
+
+# Percentage of xy-plane of the object's base aligned bbox that needs to covered by the cloth
+m.OVERLAP_AREA_PERCENTAGE = 0.5
+
+# Subsample cloth particle points to fit a convex hull for efficiency purpose
+m.N_POINTS_CONVEX_HULL = 1000
```

Comment:
Curious, what's the overall speed with using 1000 points? I'm guessing you chose this value as the empirically-validated optimum trading off fidelity and efficiency?

Reply:
overall speed with using 1000 points is ~0.5 ms. Yes I choose this value as the empirically-validated optimum trading off fidelity and efficiency.